docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -970,44 +1040,50 @@ Note that `@TestSecurity` `user` and `roles` attributes are available as `TokenI
     }
 ----
 
-This is particularly useful if the same set of security settings needs to be used in multiple test methods.
+This is particularly useful if multiple test methods must use the same set of security settings.
 ====
 
-=== How to check the errors in the logs
+=== Check log errors
```

Comment:
```suggestion
=== Check errors in the log
```

Reply:
Otherwise it reads as if we are checking `log errors` which may not be necessary related to OIDC. I have to admit I'm not sure what reads better `in the log` or `in the logs`...

Reply:
@sberyozkin - Makes sense. Thanks!